src/index.tsx

```diff
@@ -4,7 +4,7 @@ import { BrowserRouter as Router } from "react-router-dom";
 import { App } from "./App";
 
 import "bootstrap/dist/css/bootstrap.min.css";
-import "./index.scss";
+import "./styles/global.scss";
```

Comment:
question: is this just the base styling of the app basically?

Reply:
Yes, basically.